engine/sim/sc_profileset.hpp

```diff
@@ -188,11 +188,27 @@ class profile_output_data_item_t
 
 class profile_output_data_t
 {
-  race_e                                  m_race;
-  std::vector<talent_data_t*>             m_talents;
-  std::string                             m_artifact;
-  std::string                             m_crucible;
-  std::vector<profile_output_data_item_t> m_gear;
+  race_e                                       m_race;
+  std::vector<talent_data_t*>                  m_talents;
+  std::string                                  m_artifact;
+  std::string                                  m_crucible;
+  std::vector<profile_output_data_item_t>      m_gear;
+
+  double    m_crit_rating,
```

Comment:
I played around with having these stored in a different class but got in over my head with implicitly deleted copy constructors and the like.

These will be allocated if a user has any non-empty `profileset_output_data` option but it shouldn't affect the default profilesets without that option.